### PR TITLE
PXP-577: [CLI] Implement the command

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-577](https://mindvalley.atlassian.net/browse/PXP-577)

### Added
- [x] Add a new `wukong tui` command to enter TUI mode
Screenshot 2023-08-09 at 3 36 57 PM

[PXP-577]: https://mindvalley.atlassian.net/browse/PXP-577?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/130#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by onimsha** [comment]
I think we should remove the `Header` text from the layout. It should just be an empty pane at the beginning. 

For other panes, I think the pane title should be like this 

image

So that it will save some space in the pane

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/130#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/130#pullrequestreview-1570918883" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
